### ISSUE TITLE
Can't access post list actions using TalkBack

### ISSUE THREAD

**Issue by guarani** [wordpress-mobile/WordPress-Android#20770]
### Expected behavior

Using TalkBack, I should be able to open a post's three-dot menu.

### Actual behavior

I can't do this using TalkBack because I can't select the three-dot menu (it only lets me select a post (and open it in the editor).

### User report

A beta user reported this:

> How do I delete a specific article from the Articles section? I tried a lot, but I could not find the article options that previously appeared in older versions of WordPress. Note, I am a screen reader user. Thanks.

### Screen recording

https://github.com/wordpress-mobile/WordPress-Android/assets/1898325/0c1bb198-503f-48cc-b464-2cd4f9884efa

### Steps to reproduce the behavior
1. Open the Jetpack app 
2. Go to the Post list screen
3. Turn on TalkBack
4. Try to select the three-dot menu (e.g. to delete a post)
5. Try swiping or tapping to select, but notice that you can't

##### Tested on Samsung Galaxy S10, Android 12, JPAndroid 24.8 (beta)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
